src/core/library.h

```diff
@@ -203,11 +203,26 @@ typedef struct QUIC_LIBRARY {
     _Field_size_(PartitionCount)
     QUIC_PARTITION* Partitions;
 
+    //
+    // Lock protecting the stateless retry configuration.
+    //
+    CXPLAT_DISPATCH_RW_LOCK StatelessRetryLock;
+
     //
     // The base secret used to generate keys for the stateless retry token.
     //
     uint8_t BaseRetrySecret[CXPLAT_AEAD_AES_256_GCM_SIZE];
 
+    //
+    // The AEAD algorithm to use for the retry key.
+    //
+    CXPLAT_AEAD_TYPE RetryAeadAlgorithm;
+
+    //
+    // The number of milliseconds between key rotations.
+    //
+    uint32_t RetryKeyRotationMs;
```

Comment:
Are you defaulting this? Probably should be set to `QUIC_STATELESS_RETRY_KEY_LIFETIME_MS`.